### ISSUE TITLE
EmptyStackException when requiring JS bundle

### ISSUE THREAD

**Issue by aengl** [atomist/rug-cli#40]
```java.util.EmptyStackException
    at java.util.Stack.peek(Stack.java:102)
    at com.atomist.rug.compiler.typescript.DefaultSourceFileLoader.doSourceFor(DefaultSourceFileLoader.java:120)
    at com.atomist.rug.compiler.typescript.DefaultSourceFileLoader.sourceFor(DefaultSourceFileLoader.java:54)
    at com.atomist.rug.compiler.typescript.DefaultSourceFileLoader.getSourceAsStream(DefaultSourceFileLoader.java:43)
```

When using either `import * as depbundle from './depbundle'` or `const depbundle = require('./depbundle')` to import the bundle.

Project: https://github.com/blairehh/foobar-react-editors

**Comment by cdupuis** [comment]
The problem is the relative import.

**Comment by cdupuis** [comment]
This has been fixed in e6cd9e4172765b7567a564f20b0629b5865adbf9 and 793e2cca301dc05b3f650f26cc21c51292babeef
